misc/snapshotter/nydus-snapshotter.yaml

```diff
@@ -0,0 +1,191 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nydus-snapshotter-configs
+  labels:
+    app: nydus-snapshotter
+  namespace: nydus-system
+data:
+  FS_DRIVER: "fusedev"
+  ENABLE_CONFIG_FROM_VOLUME: "true"
+  ENABLE_RUNTIME_SPECIFIC_SNAPSHOTTER: "false"
+  ENABLE_SYSTEMD_SERVICE: "true"
+
+  config.toml: |-
+    version = 1
+    # Snapshotter's own home directory where it stores and creates necessary resources
+    root = "/var/lib/containerd-nydus"
+    # The snapshotter's GRPC server socket, containerd will connect to plugin on this socket
+    address = "/run/containerd-nydus/containerd-nydus-grpc.sock"
+    # The nydus daemon mode can be one of the following options: multiple, dedicated, shared, or none. 
+    # If `daemon_mode` option is not specified, the default value is multiple.
+    daemon_mode = "dedicated"
+
+    [daemon]
+    # Specify a configuration file for nydusd
+    nydusd_config = "/etc/nydus/nydusd-config.json"
+    nydusd_path = "/usr/local/bin/nydusd"
+    nydusimage_path = "/usr/local/bin/nydus-image"
+    # fusedev or fscache
+    fs_driver = "fusedev"
+
+    [log]
+    # Print logs to stdout rather than logging files
+    log_to_stdout = true
+    # Snapshotter's log level
+    level = "info"
+
+  nydusd-config.json: |-
```

Comment:
Can we reuse the `nydusd-config-xxx.json`?

Reply:
Fixed.